app/workers/fast_track/hypertension_pdf_generator.rb

```diff
@@ -0,0 +1,242 @@
+# frozen_string_literal: true
+
+module FastTrack
+  class HypertensionPdfGenerator
+    attr_accessor :patient, :bp_data, :medications
+
+    def initialize(patient, bp_data, medications, date)
+      @patient = patient
+      @bp_data = bp_data
+      @medications = medications
+      @date = date
+    end
+
+    def generate
+      pdf = Prawn::Document.new
+      pdf = add_intro(pdf)
+      pdf = add_blood_pressure(pdf)
+      pdf = add_medications(pdf) if medications.length > 1
+      add_about(pdf)
+    end
+
+    def stringify_patient
+      suffix = patient[:suffix].present? ? ", #{patient[:suffix]}" : ''
+      stringified = ''
+      %i[first middle last].each do |name|
+        stringified = "#{stringified} #{patient[name]}" if patient[name].present?
+      end
+
+      "#{stringified}#{suffix}"
+    end
+
+    def add_intro(pdf)
+      patient_name = stringify_patient
+      gen_time = Time.now.getlocal
```

Comment:
Why `.getlocal` instead of `.utc` here?

Reply:
This will be turned into a visible timestamp that will be read by US staff, so I thought a US timezone would be easier for them to parse.

Reply:
Have you tested this in your review instance? I believe the local time on production is UTC. (I'm not certain though.)

Reply:
I haven't tested this yet. The worst case right now is that the visible timestamp will be in UTC and state that it's UTC. We may tweak it to convert to a specific time zone.